src/context/AuthContext.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState, useContext, createContext } from 'react';
+
+const AuthContext = createContext(null);
+export const AuthProvider = ({ children }) => {
+  const [user, setUser] = useState('');
+  function getUsername() {
```

Comment:
- [ ] You did a good job working on your AuthContect.jsx. 💯 However, you declared a getUsername function that was never used. 🤓